config/opensearch_dashboards.yml

```diff
@@ -159,22 +159,22 @@
 #vis_type_timeline.graphiteBlockedIPs: []
 
 # opensearchDashboards.branding:
-  # logo:
-    # defaultUrl: ""
-    # darkModeUrl: ""
-  # mark:
-    # defaultUrl: ""
-    # darkModeUrl: ""
-  # loadingLogo: 
-    # defaultUrl: ""
-    # darkModeUrl: ""
-  # faviconUrl: ""
-  # applicationTitle: ""
-
-# Set the value of this setting to true to capture region blocked warnings and errors 
+# logo:
```

Comment:
how come?

Reply:
This seems to be what the linter wants

Reply:
This seems wrong. Spacing matters here, right?

Reply:
I think it works when you uncomment but while commented the linter wants them all lined up